File: typescript/docs_code_snippets/models-list.ts
import { Opper } from "opperai";

const opper = new Opper();

// List all models
const all = await opper.models.list();
for (const model of all.slice(0, 5)) {
  console.log(`${model.id} (${model.provider})`);
}

// Filter by type
const llms = await opper.models.list({ type: "llm", provider: "anthropic" });
for (const model of llms) {
  console.log(model.id);
}

File: python/docs_code_snippets/models-list.py
from opperai import Opper

opper = Opper()

# --- docs ---
# List all models
all_models = opper.models.list()
for model in all_models.models[:5]:
    print(f"{model.id} ({model.provider})")

# Filter by type
llms = opper.models.list(type="llm", provider="anthropic")
for model in llms.models:
    print(f"{model.id}")
# --- /docs ---
